src/components/UI/SearchInputLarge.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div :class="$style.inputWrapper">
+    <icon name="mdi:magnify" :size="36" :class="$style.icon" />
+    <input
+      v-model="input"
+      type="text"
+      placeholder="ユーザー検索"
+      :class="$style.input"
+      @keypress.prevent.enter.exact="submit"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref } from 'vue'
+import Icon from '/@/components/UI/Icon.vue'
+export default defineComponent({
+  name: 'SearchInputLarge',
+  components: {
+    Icon
+  },
+  setup() {
+    const input = ref<string>('')
```

Comment:
```suggestion
    const input = ref('')
```
明示する理由もないので消してよさそうです